src/weatherdata/ipm.py

```diff
@@ -141,28 +242,21 @@ def list_ressources(self):
         ---------
             dictionnary with name and description of available weatherdatasource on IPM service
         """
-        rep = self.ws.get_weatherdatasource()
+        rep = self.ipm.get_weatherdatasource()
         return {item['name']:item['description'] for item in rep}
-
+        
     def get_ressource(self, name):
```

Comment:
get_ressource() -> resource()

Reply:
How to differenciate list_resources and get_resources if i replace get_resource by resource() and list_resources by resources also